spec/index.bs

```diff
@@ -60,13 +47,10 @@ spec: ECMA-262; urlPrefix: https://tc39.es/ecma262/#
         text: agent; url: agent
         text: surrounding agent; url: surrounding-agent
         text: agent cluster; url: sec-agent-clusters
-spec: webidl; urlPrefix: https://heycam.github.io/webidl/#
-    type: dfn
-        text: resolve; url: resolve
 spec: web-apis; urlPrefix: https://html.spec.whatwg.org/multipage/webappapis.html#
     type: dfn
         text: cross-origin isolated capability; url: concept-settings-object-cross-origin-isolated-capability
-spec: canvas; urlPrefix: https://html.spec.whatwg.org/multipage/canvas.html#
+spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/canvas.html#
     type: dfn
         text: origin-clean; url: concept-canvas-origin-clean
```

Comment:
If changing this spec name to HTML (which makes sense), would it make sense to change its `urlPrefix` to `https://html.spec.whatwg.org/multipage/` and change the `url`s on the definitions to e.g. `canvas.html#concept-canvas-origin-clean`?

Reply:
Yes, that's more future-proof in case you need to add terms defined in other pages. I'll push an update.
I noticed that some of the terms defined here are actually not used in the spec. I'll drop them as well.

In an ideal world, HTML would export these terms as exported terms are those a group agrees to make available for others, whereas non-exported ones are meant to remain private and can in theory change at any time.

The same comment applies to WGSL terms by the way, which the GPU for the Web group has complete control over. Any reason not to export the WGSL terms that WebGPU needs to reference? (WGSL probably needs to be added to Bikeshed's xref database for that to work but that shouldn't be a problem)

Reply:
We'd like to be able to add cross references locally instead of through the xref database. Otherwise, I'm not sure how stable the exports from WGSL are yet, so we'd probably end up breaking our own build on a regular basis before 1.0.

This is also still super valuable post-1.0 because it would allow us to add references between the two specs in one PR instead of two.
https://github.com/tabatkins/bikeshed/issues/2006
tracked in #2709

Reply:
> We'd like to be able to add cross references locally instead of through the xref database. Otherwise, I'm not sure how stable the exports from WGSL are yet, so we'd probably end up breaking our own build on a regular basis before 1.0.
> 
> 
> 
> This is also still super valuable post-1.0 because it would allow us to add references between the two specs in one PR instead of two.
> 
> https://github.com/tabatkins/bikeshed/issues/2006
> 
> tracked in #2709